server/cli/flags.go

```diff
@@ -105,6 +105,10 @@ var flagUsage = map[string]string{
         ranges. The scan is slowed as necessary to approximately achieve this
         duration.
 `,
+	"scan-max-idle-time": `
```

Comment:
I'll leave the flags for now; we may still want to tweak things as we're testing.